src/components/tuplelang.ts

```diff
@@ -14,7 +15,7 @@ export const TUPLE_THEME_NAME = 'tuple-theme';
 export const TUPLE_DARK_THEME_NAME = 'tuple-theme-dark';
 
 export default function registerTupleLanguage(
-  monaco: any,
+  monaco: typeof monacoEditor,
```

Comment:
This type is cribbed from the typing of `useMonaco`, so this now matches what the function actually receives.